gallery/src/components/NxCombobox/NxComboboxPage.tsx

```diff
@@ -0,0 +1,283 @@
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                Callback that fires when a new search query should be performed. The <em>trimmed</em> value of the
+                text input is passed as an argument. This callback is executed whenever the user changes
+                the trimmed value of the text input, whenever the error alert's Retry button is clicked, and whenever
+                the component gains focus while in an error state (which is intended to automatically trigger a retry
+                attempt). The handler passed in for this prop should, in addition to performing the search, update
+                the <NxCode>loading</NxCode> prop. The handler receives the new value as its argument.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>autoComplete</NxCode></NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                When <NxCode>autoComplete</NxCode> is set to true, it will highlight the first matching item from the
+                array of matches and provide an inline completion string, the portion of the selected suggestion that
+                has not been typed by the user. To follow this behavior, the search logic should only return the
+                matching items that start with the search text as suggestions.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>loading</NxCode></NxTable.Cell>
+              <NxTable.Cell>boolean</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>false</NxTable.Cell>
+              <NxTable.Cell>Set to true when the search results are currently being loaded.</NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>loadError</NxCode></NxTable.Cell>
+              <NxTable.Cell>ReactNode</NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                If there is an error loading the search results, set the error message here to render an error
+                alert instead of the results within the dropdown.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>matches</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                Array of objects containing an <NxCode>id</NxCode> and a <NxCode>displayName</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>Yes</NxTable.Cell>
+              <NxTable.Cell></NxTable.Cell>
+              <NxTable.Cell>
+                The results of querying the current text input value, which are to be displayed in the
```

Comment:
This is where we should specify how the query should be done, i.e. that it should return all results that start with the text the user specified, case-insensitive.

Reply:
Updated desc here c4aafa60bb1b42bd05bd6eea7cfd69f2bf0114db.